MyPHN/app/home/home.component.ts

```diff
@@ -14,35 +16,181 @@ declare var CGSizeMake: any;
     templateUrl: "./home.component.html",
 })
 
-
 export class HomeComponent implements OnInit {
     public tabSelectedIndex: number;
     thing: number
-    constructor(private _router: Router, private locationsService: LocationsService) {
+    countyLats;
+    image: string;
+    details: LocationDetail;
+    distance;
+    name;
+    address;
+
+    constructor(
+        private _router: Router,
+        private locationsService: LocationsService,
+        private censusService: CensusService
+    ) {
         this.tabSelectedIndex = 3;
+
+
+        this.countyLats = {
+            counties: ['Ashtabula', 'Beaver', 'Blair', 'Butler', 'Cambria', 'Clarion',
+                'Crawford', 'Erie', 'Indiana', 'Jefferson', 'Lawrence', 'Mercer', 'Mifflin', 'Northumberland', 'Schuylkill', 'Trumbull', 'Warren'
+            ],
+            Ashtabula: {
+                latitude: 41.9397,
+                longitude: -80.7214
+            },
+            Beaver: {
+                latitude: 40.6368,
+                longitude: -80.3659
+            },
+            Blair: {
+                latitude: 40.4531,
+                longitude: -78.3842
+            },
+            Butler: {
+                latitude: 40.9115, longitude: -79.8297
+            },
+            Cambria: { latitude: 40.4894, longitude: -78.7476 },
+            Clarion: {
+                latitude: 41.1982, longitude: -79.4704
+            },
+            Crawford: {
+                latitude: 41.6661, longitude: -80.2767
+
+            },
+            Erie: {
+                latitude: 42.0522, longitude: -80.1875
+            },
+            Indiana: {
+                latitude: 40.6851, longitude: -79.1097
+            },
+            Jefferson: {
+                latitude: 41.1470, longitude: -78.9288
+            },
+            Lawrence: {
+                latitude: 40.9579, longitude: -80.3659
+            },
+            Mercer: {
+                latitude: 41.2620, longitude: -80.1875
+            },
+            Mifflin: {
+                latitude: 40.6523, longitude: -77.6078
+            },
+            Northumberland: {
+                latitude: 40.8674, longitude: -76.6875
+            },
+            Schuylkill: {
+                latitude: 40.6873, longitude: -76.1784
+            },
+            Trumbull: {
+                latitude: 41.3065, longitude: -80.7214
+            },
+            Warren: {
+                latitude: 41.8143, longitude: -79.2902
+            },
+        }
```

Comment:
1. Please add Westmoreland County. There have been a few location additions to the PHN network in the last couple of months, please refer to the PHN website for full list.
2. Do these lat/long coordinates refer to the center point of each county? Need clarification on how this accurately displays closest location. I can provide you with lat/long coordinates for each location if you want.